doc/relation_extraction.md

```diff
@@ -57,7 +57,7 @@ DeepDive makes good use of uncertainty to improve predictions during the
 [probabilistic inference](inference.md) step. For example, DeepDive may figure
 out that a certain mention of "Barack" is only 60% likely to actually refer to
 "Barack Obama", and use this fact to discount the impact of that mention on the
-final result for the entity "Barack Obama". DeepDive can also make use the
-domain knowledge, and allows users to encode rules such as "If Barack is married
+final result for the entity "Barack Obama". DeepDive can also make use of 
```

Comment:
No worries.  Using GitHub UI is fine, and the actual edits are more important than passing any of those silly checks.  They are just safe guards to keep the code in a clean shape easy for merging.  I think you can simply open the same file from your branch and modify inplace.  That'll probably update this PR and give us a green light.